src/components/map/MapName.js

```diff
@@ -1,28 +1,69 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
+import i18n from '@dhis2/d2-i18n';
+import InterpretationIcon from '../core/InterpretationIcon';
+import { timeFormat } from 'd3-time-format';
+
+const formatTime = timeFormat('%b %d, %Y'); // TODO: Support different locales
 
 const styles = theme => ({
-    name: {
+    root: {
+        display: 'flex',
+        justifyContent: 'center',
         position: 'absolute',
-        top: theme.spacing.unit,
-        left: '50%',
-        transform: 'translateX(-50%)',
+        maxWidth: 'calc(100% - 100px)',
+        top: 0,
+        left: 0,
+        right: 0,
+        margin: '0 auto',
         zIndex: 999,
-        padding: '6px 8px',
-        backgroundColor: 'rgba(255,255,255,0.9)',
-        boxShadow: theme.shadows[1],
-        borderRadius: theme.shape.borderRadius,
         fontSize: theme.typography.fontSize,
+        '& > div': {
+            backgroundColor: 'rgba(255,255,255,0.9)',
+            boxShadow: theme.shadows[1],
+            borderRadius: theme.shape.borderRadius,
+            margin: 8,
```

Comment:
Nitpick - can we use `theme.spacing.unit` here?

Reply:
Fixed.